R/assert.R

```diff
@@ -8,11 +8,19 @@ assert_is_character_scalar <- function(x) {
   }
 }
 
+assert_is_valid_version_label <- function(x) {
+  if (!(x %in% c("DRAFT", "APPROVED") || is.null(x))) {
+    abort("Version label must be 'DRAFT', 'APPROVED' or `NULL` but is '", x, "'.")
```

Comment:
not stop?

Reply:
not a hard stop, as it will break the workflow